api/src/legacy_soap_api/grantors/services/get_submission_list_expanded_response.py

```diff
@@ -83,18 +86,48 @@ def get_submission_list_expanded_response(
             )
         )
     )
+
     grants_gov_tracking_numbers = None
     cfda_numbers = None
     funding_opportunity_numbers = None
+    opportunity_ids = None
+    competition_ids = None
+    package_ids = None
+    status = None
+    simpler_status = None
+    submission_titles = None
     if get_submission_list_expanded_request.expanded_application_filter:
         if (
             submission_filters := get_submission_list_expanded_request.expanded_application_filter.filters
         ):
             grants_gov_tracking_numbers = submission_filters.get("GrantsGovTrackingNumber")
             cfda_numbers = submission_filters.get("CFDANumber")
             funding_opportunity_numbers = submission_filters.get("FundingOpportunityNumber")
+            opportunity_ids = submission_filters.get("OpportunityID")
+            competition_ids = submission_filters.get("CompetitionID")
+            package_ids = submission_filters.get("PackageID")
+            status = submission_filters.get("Status")
+            submission_titles = submission_filters.get("SubmissionTitle")
+
+    STATUS_TRANSFORM = {
+        "Receiving": None,
+        "Received": None,
+        "Processing": ApplicationStatus.SUBMITTED,
+        "Validated": ApplicationStatus.ACCEPTED,
+        "Rejected with Errors": None,
+        "Download Preparation": None,
+        "Received by Agency": ApplicationStatus.ACCEPTED,
+        "Agency Tracking Number Assigned": ApplicationStatus.ACCEPTED,
+    }
+    # Getting only one status from a potential list of statuses because multiple statuses on a filter don't make sense'
+    if isinstance(status, list) and len(status) > 0:
+        simpler_status = STATUS_TRANSFORM.get(status[0]) if isinstance(status[0], str) else None
+    else:
+        simpler_status = None
```

Comment:
How does grants.gov respond to multiple statuses, it would make sense if it's an OR (a list being "Status A" or "Status B"). If their logic is conceptually "A AND B" then I'd almost say the appropriate thing is to immediately skip everything else.